code/lib/preview-api/src/modules/store/csf/prepareStory.ts

```diff
@@ -149,6 +154,25 @@ export function prepareStory<TRenderer extends Renderer>(
   };
 }
 
+export const getTriangularNumber = (seed: number) => (seed * (seed + 1)) / 2;
+
+export const delayForCanvas = async (canvasElement: HTMLElement) => {
+  let backOff = 0;
+  while (!canvasElement.innerHTML) {
+    backOff += 1;
+    if (backOff > 100) {
```

Comment:
`getTriangularNumber(100)` yields 5050. 5 seconds seems like a lot given I only needed 1, and others on the issue didn't even need that.